src/sentry/integrations/pagerduty/notify_action.py

```diff
@@ -44,24 +46,38 @@ def clean(self):
                 raise forms.ValidationError(_("Invalid account"), code="invalid")
 
         service_id = cleaned_data.get("service")
+        if service_id:
```

Comment:
Can this be an early return / guard statment instead?

Reply:
I can't return early cleanly here but I can extract parts of this into helper method and it'll be clearer.